pkg/acsengine/pki.go

```diff
@@ -68,6 +72,18 @@ func CreatePki(extraFQDNs []string, extraIPs []net.IP, clusterDomain string, caP
 
 	go func() {
 		var err error
```

Comment:
Thanks for reviewing. I think we don't need extraFQDN on etcd server. IPs is enough since etcd server endpoint is not exposed outside the cluster. In my change I use the IP address of the etcd server as the common name and replaced the 127.0.0.1 loopback address in api server.